### ISSUE TITLE
Content of the main section is not adapting to the high contrast aquatic & desert modes present in the report pages.

### ISSUE THREAD

**Issue by PiJoCoder** [microsoft/SqlNexus#338]
Note: Same issue replicate for all pages (Reports) in the application.

Impact on a Person with Disability (PWD):
Color sensitive users are unable to read the content present on the page, as content present on the page is not adapting to the High contrast black modes. Due to this issue users are not able to get the information about the content present in the main section.

Test Environment:​
OS: Windows 11 [Version 22H2 (OS Build 22621.2283)].

Desktop application: SQL Nexus.

Themes: Aquatic/Desert Modes.

Pre-Requisites:
1. Install SQL NEXUS application on the desktop.
2. Settings->Accessibility ->Contrast theme mode-> (Aquatic/Desert) Modes.

Repro Steps:
Step 1: Open the SQL Nexus app in desktop.
Step 2: Connect to server Dialog gets displayed. Connect to the server by invoking connect button.
Step3: 'SQL Pref Main' page getting displayed.
Step 4: Navigate to the main section in the page using tab/shift tab keys.
Step 5: Verify whether Content of the main section is adapting to the high contrast aquatic & desert modes present in the 'SQL Pref Main' page or not.
 
Actual Result:
Content of the main section is not adapting to the high contrast aquatic & desert modes present in the 'SQL Pref Main' page.
Observation: 
1. Only toolbar section present in the header is adapting to the high contrast aquatic & desert modes.

Expected Result:
Content of the main section should be adapting to the high contrast aquatic & desert modes present in the 'Main' tab page.
Ex: All the content present on the page should be adapting to the high contrast aquatic & desert modes

Suggestive Fixes:
Check your CSS: Ensure that your CSS is not overriding the default link styles in high contrast mode1. If you’re using a framework or library, make sure it supports high contrast mode1.
Use semantic HTML: Make sure you’re using the <a> tag for links. Some browsers and assistive technologies have specific behavior for this tag, which can help ensure links are displayed correctly2.
Adjust color settings: You can adjust the color settings for hyperlinks in your high contrast theme. To do this, go to Settings > Ease of Access > High contrast, then select a color or enter a hex code for 'Hyperlinks’3.
Use forced-colors CSS media query: The forced-colors CSS media query allows you to detect if a user agent has forced colors into a restricted palette. You can use this to apply specific styles when high contrast mode is active2.
Test your changes: After making these changes, test your website with high contrast mode enabled to ensure that links are displayed correctly2.

**Comment by JamesFerebee** [comment]
171163

**Comment by PiJoCoder** [comment]
- [ ] Active Queries_C.rdl  - In Progress
- [ ] Active Traces and XEvents_C - Backup.rdl
- [ ] Active Traces and XEvents_C.rdl
- [ ] AlwaysOn_AGBasics_C.rdl
- [ ] AlwaysOn_AGDetails_C.rdl
- [ ] AnalysisSummary_C.rdl
- [ ] Blocking and Wait Statistics_C.rdl
- [ ] Blocking Chain Detail_C.rdl
- [ ] Blocking Runtime Detail_C.rdl
- [ ] Bottleneck Analysis_C.rdl
- [ ] DatabaseConfiguration_C.rdl
- [ ] Errors and Warnings_C.rdl
- [ ] Filter_Drivers_C.rdl
- [ ] Instructions.rdl
- [ ] Linux_Perfmon_C.rdl
- [ ] Linux_Perfmon_CPU_C.rdl
- [ ] Linux_Perfmon_IO_C.rdl
- [ ] Linux_Perfmon_Memory_C.rdl
- [ ] Linux_Perfmon_Network_C.rdl
- [ ] Loaded Modules_C.rdl
- [ ] LockSummary_C.rdl
- [ ] Memory Brokers_C.rdl
- [ ] Memory Clerks_C.rdl
- [ ] Missing Indexes_C.rdl
- [ ] Other Waits_C.rdl
- [ ] PAL_C.rdl
- [ ] PAL_sub_C.rdl
- [ ] Perfmon_C - Backup.rdl
- [ ] Perfmon_C.rdl
- [ ] Perfmon_CPU_C.rdl
- [ ] Perfmon_IO_C.rdl
- [ ] Perfmon_Memory_C.rdl
- [ ] Perfmon_Network_C.rdl
- [ ] Query Execution Memory Details_C.rdl
- [ ] Query Execution Memory_C.rdl
- [ ] Query Hash_C.rdl
- [ ] Query_Store_C.rdl
- [ ] Query_Store_Details_C.rdl
- [ ] Replication_Topology_C.rdl
- [ ] ServerConfiguration_C.rdl
- [ ] Spinlock Details_C.rdl
- [ ] Spinlock Stats_C.rdl
- [ ] SQL 2000 Blocking Detail_C.rdl
- [ ] SQL 2000 Blocking_C.rdl
- [ ] SQL Perf Main.rdl
- [ ] SysIndexes_C.rdl
- [ ] Sysprocesses_C.rdl
- [ ] System_Requests_C.rdl
- [ ] Tempdb_Space_Use_C.rdl
- [ ] TopN_Resources_By_QDS.rdl
- [ ] TopPlanAnalysis_C.rdl
- [ ] Virtual File Stats Details_C.rdl
- [ ] Virtual File Stats_C.rdl
- [ ] WaitDetails_C.rdl
- [ ] WASD_Connection_Stats_C.rdl
- [ ] WASD_Deadlocks_C.rdl
- [ ] WASD_Event_Log_C.rdl
- [ ] WASD_ResourceStats_C.rdl
- [ ] WASD_ResourceUsage_C.rdl
- [ ] Working Set Trim_C.rdl

**Comment by PiJoCoder** [comment]
This is now all resolved by PR #492
